ci-operator/config/redhat-performance/jetlag/redhat-performance-jetlag-main.yaml

```diff
@@ -56,6 +56,19 @@ tests:
     env:
       NUM_WORKER_NODES: "0"
     workflow: openshift-qe-installer-bm-deploy
+- always_run: false
+  as: deploy-mno-scaleout
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perfscale-jetlag
+    env:
+      NUM_WORKER_NODES: "0"
```

Comment:
I think it would be better to start with 1 and end with 2, to avoid the strange labeling of compact clusters